core/irCheck.ml

```diff
@@ -745,6 +745,63 @@ struct
 
               Query (range, e, t), t, o
 
+        | InsertRows (source, rows)
+	| InsertReturning (source, rows, _) ->
```

Comment:
We *could* unify these into a single constructor, with the `returning` an option, which is what the source AST does.  I did it this way because the translation from the previous library functions to `evalir` cases is more direct.  But it might actually be better/cleaner to do it the other way.